docs/aws/audit/redshiftmonitoring/rules/emr_kerberos_enabled.mdx

```diff
@@ -23,6 +23,260 @@ CBP,RBI_MD_ITF
+   - Open the Amazon EMR console at [https://console.aws.amazon.com/elasticmapreduce/](https://console.aws.amazon.com/elasticmapreduce/).
+
+2. **Create or Edit a Security Configuration:**
+   - In the left-hand navigation pane, choose **Security configurations**.
+   - Click on **Create security configuration** or select an existing security configuration to edit.
+   - Under **Kerberos authentication**, check the box to enable Kerberos authentication.
+   - Provide the necessary details for the Kerberos configuration, such as the KDC (Key Distribution Center) and realm information.
+
+3. **Apply the Security Configuration to Your EMR Cluster:**
+   - When creating a new EMR cluster, in the **Security and access** section, select the security configuration that has Kerberos enabled.
+   - If editing an existing cluster, ensure that the cluster is terminated and then create a new cluster with the appropriate security configuration.
+
+4. **Review and Launch:**
+   - Review all the settings to ensure Kerberos is enabled.
+   - Click **Create cluster** to launch the EMR cluster with the Kerberos-enabled security configuration.
+
+By following these steps, you can ensure that your Amazon EMR clusters have Kerberos enabled, enhancing the security of your data and operations.
+</Accordion>
+
+<Accordion title='Using CLI'>
+To prevent Amazon EMR clusters from being created without Kerberos enabled using AWS CLI, you can follow these steps:
+
+1. **Create a Security Configuration with Kerberos Enabled:**
+   First, create a security configuration that enables Kerberos. This configuration will be used when creating EMR clusters.
+
+   ```sh
+   aws emr create-security-configuration --name MyKerberosConfig --security-configuration '{
+     "AuthenticationConfiguration": {
+       "KerberosConfiguration": {
+         "Provider": "ClusterDedicatedKdc",
+         "ClusterDedicatedKdcConfiguration": {
+           "TicketLifetimeInHours": 24,
+           "CrossRealmTrustPrincipalPassword": "YourPassword"
+         }
+       }
+     }
+   }'
+   ```
+
+2. **Create an EMR Cluster with the Security Configuration:**
+   When creating an EMR cluster, specify the security configuration that you created in the previous step.
+
+   ```sh
+   aws emr create-cluster --name "MyCluster" --release-label emr-5.30.0 --applications Name=Hadoop Name=Spark --ec2-attributes KeyName=MyKey --instance-type m5.xlarge --instance-count 3 --use-default-roles --security-configuration MyKerberosConfig
+   ```
+
+3. **Set Up IAM Policies to Enforce Security Configuration:**
+   Ensure that IAM policies enforce the use of the security configuration when creating EMR clusters. This can be done by creating a policy that allows only the creation of clusters with the specified security configuration.
+
+   ```sh
+   aws iam create-policy --policy-name EnforceKerberosPolicy --policy-document '{
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Deny",
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>